_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -40067,7 +40065,6 @@
 /obj/structure/bed/medical/emergency,
 /obj/structure/bed/medical/emergency,
 /obj/machinery/iv_drip,
-/obj/machinery/iv_drip,
```

Comment:
Looks like this IV drip was intentionally stacked considering there's also two beds.

Reply:
Right now the linter does not support excluding a subtype in conjunction with identical.  This may be the only `obj/machinery` where this might be a grey area. I'd recommend we just not allow it entirely.  (and also iv drips should really be a subtype of `/obj/structure/`)

Either we delete it, or move the 2nd iv drip a tile over.

Reply:
Well the intent seems to be 2 iv drips, 2 roller beds, and 2 stationary beds. 
So maybe just move the IV drips overtop the stationary beds rather than the roller beds? 

> Do we want to do this? I don't think you can drag one IV stand onto another.

Yeah, you can, they're non-dense.

Reply:
![StrongDMM_naRvAlf6HA](https://github.com/tgstation/tgstation/assets/5195984/19a732f5-38e1-4fc8-9f30-22f3497627f8)

Moved the the pairs of straightjacket, blindfold, and muzzle to be on the orginal iv drip tile.